Gemfile

```diff
@@ -13,6 +13,7 @@ gem 'dotenv-rails', '~> 2.7', '>= 2.7.6' # https://rubygems.org/gems/dotenv-rail
 gem 'jsonapi-serializer', '~> 2.2' # https://rubygems.org/gems/jsonapi-serializer
 gem 'pg', '~> 1.1' # https://rubygems.org/gems/pg
 gem 'puma', '~> 5.0' # https://rubygems.org/gems/puma
+gem 'rack', '~> 2.2', '>= 2.2.3' # https://rubygems.org/gems/rack
```

Comment:
I'm not sure in this case, but `rack` is a dependency of `rails`(maybe don't need to include it in Gemfile).